testing-guidance/workflow-tests/animation/testassets/example-asset-screenshots.md

```diff
@@ -0,0 +1,26 @@
+# Purpose
+This reference file is intended to show what example assets looked like when generated. The reason to do this is that systems can be updated causing example assets to go out of date. Having screenshots of their setup allows for rebuilding of assets in new formats when upgrading is not possible.
```

Comment:
Changing *generated* to *created* would avoid potential confusion, as these assets were created manually following instruction/improvising and not generating them.

*This reference file is intended to show what example assets looked like when **generated**.* > *This reference file is intended to show what example assets looked like when **created**.*

Reply:
Changed generated to created.